tools/crm_resource_runtime.c

```diff
@@ -1552,6 +1552,8 @@ cli_resource_execute(const char *rsc_id, const char *rsc_action, GHashTable *ove
                    action, rsc->id, rclass, rprov ? rprov : "", rtype, op->status);
         }
 
+        if(!do_trace) goto done;
+
         if (op->stdout_data) {
```

Comment:
I'd rather not force verbose on all other commands. I think the easiest approach would be to add here:

    && (do_trace || safe_str_neq(rsc_action, "validate-all")